locations/jclouds/src/main/java/brooklyn/location/jclouds/JcloudsLocation.java

```diff
@@ -938,7 +1058,11 @@ public void apply(TemplateOptions t, ConfigBag props, Object v) {
                     public void apply(TemplateOptions t, ConfigBag props, Object v) {
                         if (t instanceof EC2TemplateOptions) {
                             if (v==null) return;
-                            ((EC2TemplateOptions)t).userData(v.toString().getBytes());
+                            String data = v.toString();
+                            if (!(data.startsWith("<script>") || data.startsWith("<powershell>"))) {
+                                data = "<script> " + data + " </script>";
```

Comment:
Do we have a test that this works for linux and for windows? (haven't reached your tests in the PR review yet).

Out of interest, where in jclouds is this documented?